src/commandlineflags.cc

```diff
@@ -216,4 +209,34 @@ bool ParseStringFlag(const char* str, const char* flag, std::string* value) {
 bool IsFlag(const char* str, const char* flag) {
   return (ParseFlagValue(str, flag, true) != nullptr);
 }
+
+// If |*pstr| begins with |prefix|, advance *pstr so that it points
+// just past the prefix, and return true. Otherwise, return false.
+static bool SkipPrefix(const char* prefix, const char** pstr) {
```

Comment:
we don't need HasBenchmarkFlagPrefix (see earlier comment) and SkipPrefix is only used by it.

For colored output, this is not necessary.